Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -3967,6 +3967,92 @@ def check_skip_filter(checkid, font=None, **iterargs):
   return True, None
 
 
+def typo_metrics_enabled(ttfont):
```

Comment:
our code style has been to use `ttFont` to denote an instance of `TTFont`

Reply:
also, is this just a helper function or is it a condition?

Reply:
It's just a helper function.